resources/assets/js/videos/components/videoScreen/drawInteractions.vue

```diff
@@ -217,6 +237,22 @@ export default {
             }
 
             this.$emit('pending-annotation', this.pendingAnnotation);
+
+            if (this.singleAnnotation) {
+                if (this.isDrawingPoint) {
+                    if (this.isPointDoubleClick(e)) {
+                        this.resetPendingAnnotation(this.pendingAnnotation.shape);
+                        return;
+                    }
+                    this.lastDrawnPointTime = new Date().getTime();
+                    this.lastDrawnPoint = e.feature.getGeometry();
+                }
+                this.pendingAnnotationSource.once('addfeature', this.finishDrawAnnotation);
+            }
```

Comment:
I feel all this can be added to the `if` above when refactored appropriately. This also would add the ´return` I asked for above.